doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -154,7 +156,7 @@ Architecture they must be implemented as per the following specifications:
 |`ra2.ntw.004`|Multiplexer/meta-plugin CNI Plugins|The selected multiplexer/meta-plugin must support the use of multiple CNI-conformant Network Plugins.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.ntw.005`|SR-IOV Device Plugin for Network Intensive|When hosting workloads matching the Network Intensive profile, a Device Plugin for SR-IOV must be used.|[`e.cap.013`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
 |`ra2.ntw.006`|SR-IOV Network Plugin for Network Intensive|When hosting workloads matching the Network Intensive profile, a CNI-conformant Network Plugin for SR-IOV must be used.|[`e.cap.013`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
-|`ra2.ntw.007`|User plane networking|When hosting workloads matching the Network Intensive profile, a CNI network plugin that supports the use of OVS-DPDK or VPP vSwitches must be deployed as part of the networking solution.|[`infra.net.acc.cfg.001`](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
+|`ra2.ntw.007`|User plane overlay networking|When hosting workloads matching the Network Intensive profile, a CNI network plugin that supports the use of OVS-DPDK or VPP vSwitches should be deployed as part of the networking solution.|[`infra.net.acc.cfg.001`](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
```

Comment:
@rgs the original requirement states this is a must for DPDK:
![image](https://user-images.githubusercontent.com/52716470/102067588-37f3bd80-3df3-11eb-9520-5bd0f4be3778.png)

Reply:
i just noticed that note (3) under infra.net.acc.cfg.001 states:

> this feature is not applicable for Kubernetes based infrastructures due to lack of vSwitch however workloads need access to user space networking solutions.

I'm not sure about it, many CNIs (and NSM) have a dataplane component with a vswitch running.

Realistically, an user of a network intensive CNF would use either SRIOV or a vswitch-based CNI with some sort of acceleration (eg DPDK), but probably not both at the same time. we can reword this as a "must", and the SR-IOV one as a "may" or "should", or viceversa.

Reply:
yeah makes sense - we should have an issue to update both the RM and the RA2 ch2 (this `infra.net.acc.cfg.001` is sourced in the RM chapter 5).

Reply:
@rgstori You may have to word it so that they are exclusive -- both can be "must" but the exclusivity would ensure support for only one. Should this be a host (node) level?

Reply:
agreed to have a single "MUST" requirement, that has all 3 options: sriov, dpdk, vpp etc.

Reply:
@rgstori @pgoyal01 are either of you able to prepare a change to the PR to accommodate this?

Reply:
edited in https://github.com/cntt-n/CNTT/commit/5c28f9fd19328d3452e1a9618be181416f1b8cc1 . the requirement is ra2.ntw.010 now

Reply:
ra2.ntw.008 also mentions sr-iov for network intensive applications, have changed its verb to 'may'